Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG CONTAINERD_VERSION=v1.7.18
 ARG CONTAINERD_ALT_VERSION_16=v1.6.33
 ARG REGISTRY_VERSION=v2.8.3
 ARG ROOTLESSKIT_VERSION=v2.0.0
-ARG CNI_VERSION=v1.3.0
+ARG CNI_VERSION=v1.5.0
 ARG STARGZ_SNAPSHOTTER_VERSION=v0.15.1
 ARG NERDCTL_VERSION=v1.6.2
 ARG DNSNAME_VERSION=v1.3.1
```

Comment:
@vito Looks like dnsname has been archived and development has moved to other repos. Could you update this feature when you get a chance